posts/2020/07/the-ibis-backends.md

```diff
@@ -0,0 +1,411 @@
+
+
+
+
+<div>
+<style scoped>
+    .dataframe tbody tr th:only-of-type {
+        vertical-align: middle;
+    }
+
+    .dataframe tbody tr th {
+        vertical-align: top;
+    }
+
+    .dataframe thead th {
+        text-align: right;
+    }
+</style>
+<table border="1" class="dataframe">
+  <thead>
+    <tr style="text-align: right;">
+      <th></th>
+      <th>omnisci</th>
+      <th>spark</th>
+      <th>postgres</th>
+      <th>bigquery</th>
+      <th>pandas</th>
+      <th>sqlite</th>
+      <th>impala</th>
+      <th>kudu</th>
+      <th>geospatial</th>
+      <th>clickhouse</th>
+      <th>mysql</th>
+      <th>sqlalchemy</th>
+    </tr>
+    <tr>
+      <th>year</th>
+      <th></th>
+      <th></th>
+      <th></th>
+      <th></th>
+      <th></th>
+      <th></th>
+      <th></th>
+      <th></th>
+      <th></th>
+      <th></th>
+      <th></th>
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>2015</th>
+      <td></td>
+      <td></td>
+      <td>2</td>
+      <td></td>
+      <td>2</td>
+      <td>25</td>
+      <td>52</td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td>17</td>
+    </tr>
+    <tr>
+      <th>2016</th>
+      <td></td>
+      <td></td>
+      <td>3</td>
+      <td></td>
+      <td></td>
+      <td>2</td>
+      <td>4</td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td>3</td>
+    </tr>
+    <tr>
+      <th>2017</th>
+      <td></td>
+      <td>1</td>
+      <td>21</td>
+      <td>15</td>
+      <td>49</td>
+      <td>10</td>
+      <td>15</td>
+      <td></td>
+      <td></td>
+      <td>8</td>
+      <td></td>
+      <td>10</td>
+    </tr>
+    <tr>
+      <th>2018</th>
+      <td>31</td>
+      <td></td>
+      <td>10</td>
+      <td>71</td>
+      <td>35</td>
+      <td>8</td>
+      <td>17</td>
+      <td></td>
+      <td></td>
+      <td>9</td>
+      <td>2</td>
+      <td>2</td>
+    </tr>
+    <tr>
+      <th>2019</th>
+      <td>33</td>
+      <td>22</td>
+      <td>17</td>
+      <td>12</td>
+      <td>32</td>
+      <td>1</td>
+      <td>4</td>
+      <td></td>
+      <td>7</td>
+      <td>1</td>
+      <td>2</td>
+      <td>5</td>
+    </tr>
+    <tr>
+      <th>2020</th>
+      <td>38</td>
+      <td>3</td>
+      <td>4</td>
+      <td>2</td>
+      <td>4</td>
+      <td>1</td>
+      <td>2</td>
+      <td>1</td>
+      <td>3</td>
+      <td>4</td>
+      <td>4</td>
+      <td></td>
+    </tr>
+  </tbody>
+</table>
+</div>
+
+
+> We note an early focus `ibis.sqlite, sqlalchemy and ibis.impala`. 
+Later, work began on the `pandas` backend rounding out the three different types of backgrounds.
+From this point, improvements were made to these key backends as `ibis.clickhouse, ibis.spark and "postgres"`. Recently, 
+For the past 3 years, Quansight, in partnership with OmniSci, added the `ibis.omniscidb`
+string generating backend. Since Quansight Labs has taken on a role as a community maintainer 
```

Comment:
[Since Quansight Labs has taken on a role as a community maintainer ] incomplete sentence. I'm not sure how to work this in without downplaying the partnership with Omnisci. But I can see how noting both are important.